pyrsia_node/src/main.rs

```diff
@@ -61,6 +62,8 @@ use std::{
 use tokio::io::{self, AsyncBufReadExt};
 use tokio::sync::Mutex;
 use warp::Filter;
+use crate::block_chain::block::Block;
+use crate::block_chain::block_chain::BlockChain;
```

Comment:
I think the `crate` is extra since we have the `mod` 🤔 

```suggestion
use block_chain::{block::Block, block_chain::BlockChain};
```

Reply:
It wouldn't compile without, and my editor suggested it. I'll try again

Reply:
```
215 |     let mut bc = BlockChain::new();
    |                  ^^^^^^^^^^ not found in this scope
    |
help: consider importing this struct
    |
37  | use crate::block_chain::block_chain::BlockChain;
    |

error[E0422]: cannot find struct, variant or union type `Block` in this scope
   --> pyrsia_node/src/main.rs:259:37
    |
259 |                         let block = Block{
    |                                     ^^^^^ not found in this scope
    |

```
When I do as you suggest

Reply:
Your right.

I opened a PR but did not change my suggestion here... correct code is https://github.com/chb0github/pyrsia/pull/1/files#diff-2c5d9cbb9033cbd429ad804a903684afdac3c2f5cb96bd0c474b56f0793521fbR39-R40

Thanks for running CI, it passed https://github.com/chb0github/pyrsia/actions/runs/1594267461 🚀